src/wovnio/utils/HTTPHeaderParser.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace Wovnio\Utils\HTTPHeaderParser;
+
+class HTTPHeaderParser
+{
+    public static function parseRawHeader($rawHeader)
+    {
+        if (!$rawHeader) {
+            return array();
+        }
+
+        $rawHeader = explode("\r\n", $rawHeader);
+        $parsedHeaders = array('status' => $rawHeader[0]);
+        foreach ($rawHeader as $value) {
+            $exploded = explode(':', $value, 2);
+            if (sizeof($exploded) == 2) {
+                $parsedHeaders[trim($exploded[0])] = trim($exploded[1]);
+            }
+        }
+        return $parsedHeaders;
+    }
+
+    public static function parseRawResponse($rawResponse, $headerSize)
```

Comment:
This function seems not to be used.

Reply:
Yep it is not used at the moment. But I choose to keep it just in case we need to parse a raw response in the future. And this function sort of serves the purpose of explaining where the $rawHeader in `parseRawHeader()` should come from.

Reply:
I feel this function is almost same as the following line.
So, this function can be replaced by this line.
```
$parsedHeaders = HTTPHeaderParser::parseRawHeader(substr($response, 0, $header_size));
```

Reply:
I replaced this line with the use of that previously unused util function `parseRawResponse()`.